Sources/StructuredQueries/Macros.swift

```diff
@@ -34,11 +34,13 @@ public macro Table(_ name: String? = nil) =
 ///   - representableType: A type that represents the property type in a query expression. For types
 ///     that don't have a single representation in SQL, like `Date` and `UUID`.
 ///   - primaryKey: The column is its table's auto-incrementing primary key.
+///   - databaseInitialized: The column has a default value and is not needed in an insert statement.
 @attached(accessor, names: named(willSet))
 public macro Column(
   _ name: String? = nil,
   as representableType: (any QueryRepresentable.Type)? = nil,
-  primaryKey: Bool = false
+  primaryKey: Bool? = nil,
+  databaseInitialized: Bool? = nil
```

Comment:
In the spirit of the above comment, I think these parameters could stay `Bool` and not be optional, which would further simplify diagnostics added to the macro.